src/scenes/home/header/sideNav/sideNav.css

```diff
@@ -3,7 +3,6 @@
   background-color:rgba(47,52,72,.95);
   height: 100%;
   width: 100%;
-  position: fixed;
 }
 
 .header {
```

Comment:
Change .header {...} to
```
.header {
  display: inline-block;
  line-height: 0;
}
```
Set .close top to 27px and .logo top to 25px

Delete .logoWrapper styles entirely.